src/components/DiffView/index.d.ts

```diff
@@ -0,0 +1,38 @@
+declare module 'react-diff-view' {
+  export interface HunkInfo {
+    content: string;
+    oldStart: number;
+    newStart: number;
+    oldLines: number;
+    newLines: number;
+    changes: Array<object>;
```

Comment:
Can we use something else than `object` (a more precise type?)

Reply:
I commented on this in the comments I added with my latest push. I don't believe we ever access this `changes` prop in our app. It's just something that is embedded in a `Hunk`, so I wasn't sure that it made sense to fully document it. I suppose, on the other hand, we don't know that we'll *never* make use of it, so maybe I should fully document it? Is that what you suggest?